tables/isolate_mutations.csv

```diff
@@ -12954,16 +12954,12 @@ BA.1 RBD,S,496,S,NULL,NULL
 BA.1 RBD,S,498,R,NULL,NULL
 BA.1 RBD,S,501,Y,NULL,NULL
 BA.1 RBD,S,505,H,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,80,A,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,215,G,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,241,del,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,242,del,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,243,del,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,417,N,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,484,K,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,501,Y,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,614,G,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,655,Y,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,701,V,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,1027,I,NULL,NULL
-B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,1176,F,NULL,NULL
+B.1.351 Spike:-18F+215G+241del-244del-246I,S,80,A,NULL,NULL
+B.1.351 Spike:-18F+215G+241del-244del-246I,S,241,del,NULL,NULL

```

Comment:
The 241-243del and 242-244del are equivalent since 241 and 244 are both leucine. The canonical form should be 241-243del, as the same as our Sierra-SARS2 program and Outbreak.info.

Reply:
Ah, good to know. I wasn't sure how to ask about that